rust-checks/src/bin/no-python-tests.rs

```diff
@@ -0,0 +1,29 @@
+use nixpkgs_hammering_ast_checks::analysis::*;
+use nixpkgs_hammering_ast_checks::common_structs::*;
+use regex::Regex;
+use std::error::Error;
+use std::io::{self, BufRead, BufReader};
+use std::process::ChildStdout;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let attrs: Vec<Attr> = serde_json::from_reader(io::stdin())?;
+    println!("{}", analyze_log_files(attrs, analyze_single_file)?);
+    Ok(())
+}
+
+fn analyze_single_file(log: BufReader<ChildStdout>) -> Result<Report, Box<dyn Error>> {
+    let re = Regex::new(r"Ran 0 tests in 0.000s").unwrap();
+
+    let report = match log.lines().any(|l| re.is_match(&l.unwrap())) {
+        true => vec![NixpkgsHammerMessage {
+            msg: "Zero tests run by pytest detected: 'Ran 0 tests in 0.000s'"
```

Comment:
Is not pytest just one if the runners? Maybe something like “Test runner could not discover any test cases” would be more suitable.